docs/docusaurus.config.js

```diff
@@ -155,9 +158,9 @@ module.exports = {
           showLastUpdateAuthor: true,
           showLastUpdateTime: true,
           remarkPlugins: [
+            npm2yarn2pnpm,
```

Comment:
Can we do `require('@sapphire/docusaurus-plugin-npm2yarn2pnpm')` here to make it consistent with the rest of the array? 🤔

Reply:
No unfortunately not, that was the whole issue. It needs to be destructured. 

Maybe `require('@sapphire...').npm2yarn2pnpm` would work though? Idk

Reply:
`require('@sapphire...').npm2yarn2pnpm` did seem to work :tada: